formfactory/views.py

```diff
@@ -53,17 +73,26 @@ def get_prefix(self):
         return self.kwargs.get("slug", self.form_slug)
 
     def get_success_url(self):
+
+        # If the intial post was an AJAX call we append the ajax query to the
+        # url. Primarily for when we return another formfactory view that
+        # already has some logic for AJAX.
+        ajax = self.request.GET.get("ajax")
+        url = "%s"
+        if ajax == "true":
+            url = "%s?ajax=true"
         redirect_url = self.form_object.redirect_to or self.request.GET.get(
             SETTINGS["redirect-url-param-name"]) or self.redirect_to
         if redirect_url:
-            return redirect_url
-        return self.request.path_info
+            return url % redirect_url
+        return url % self.request.path_info
 
     def get_context_data(self, **kwargs):
         context = super(FactoryFormView, self).get_context_data(**kwargs)
         context.update({
-            "form_object": self.form_object
+            "object": self.form_object
```

Comment:
There's code in production that refers to `form_object`. We can only merge this as is once the code code has been versioned and released.